src/client/utils/dom.js

```diff
@@ -456,24 +452,45 @@ export function isRenderedNode (node) {
     return !(isProcessingInstructionNode(node) || isCommentNode(node) || /^(script|style)$/i.test(node.nodeName));
 }
 
+export function getTabIndex (el) {
+    // NOTE: we get tabIndex value by getting attribute because el.tabIndex property
+    // returns -1 for some elements (e.g. for body) for not assigned tabIndex
```

Comment:
> for not assigned tabIndex

what does it mean? elements that do not have a tabIndex assigned? because these elements cannot be assigned a tabIndex?

If the former, than

```
    // NOTE: we obtain the tabIndex value from an attribute because the el.tabIndex property
    // returns -1 for some elements (e.g. for body) with no tabIndex assigned.
```